### PR TITLE
docs: Update security considerations for wasm file pickers

### PR DESCRIPTION
**Pull request by jeromelaban**

### CODE REVIEW

File: doc/articles/features/windows-storage-pickers.md [hunk 1]
```diff
@@ -263,6 +263,13 @@ The allowed values for the configuration are:
 - `FileSystemAccessApi` - uses File System Access API only. If not available, pickers will throw `NotSupportedException`
 - `DownloadUpload` - uses download/upload pickers only.
 
+### Security considerations
```
Comment (Youssef1313) [inline doc/articles/features/windows-storage-pickers.md:266]: nit:

```suggestion
### Security considerations

```

Markdownlint prefers surrounding headers with blank lines.

File: doc/articles/features/windows-storage-pickers.md [hunk 2]
```diff
@@ -263,6 +263,13 @@ The allowed values for the configuration are:
...
+Browsers generally treat file opening/save operations as sensitive operations, and the following message may appear when using this APIs:
+```
+SecurityError: Failed to execute 'showSaveFilePicker' on 'Window': Must be handling a user gesture to show a file picker.
+```
```
Comment (Youssef1313) [inline doc/articles/features/windows-storage-pickers.md:270]:
~~~suggestion

```
SecurityError: Failed to execute 'showSaveFilePicker' on 'Window': Must be handling a user gesture to show a file picker.
```

~~~

File: doc/articles/features/windows-storage-pickers.md (hunk 1 above)
Comment (jeromelaban) [reply doc/articles/features/windows-storage-pickers.md:266]: Thanks !

File: doc/articles/features/windows-storage-pickers.md [hunk 3]
```diff
@@ -263,6 +263,15 @@ The allowed values for the configuration are:
 - `FileSystemAccessApi` - uses File System Access API only. If not available, pickers will throw `NotSupportedException`
 - `DownloadUpload` - uses download/upload pickers only.
 
+### Security considerations
+Browsers generally treat file opening/save operations as sensitive operations, and the following message may appear when using this APIs:
```
Comment (Xiaoy312) [inline doc/articles/features/windows-storage-pickers.md:267]:
```suggestion
Browsers generally treat file opening/saving operations as sensitive operations, and the following message may appear when using this APIs:
```

**Review by Xiaoy312 — APPROVED** [review]

**Pull request merged**